snuba/migrations/snuba_migrations/metrics/0006_metrics_distributions.py

```diff
@@ -2,7 +2,7 @@
 
 from snuba.migrations import migration, operations
 from snuba.migrations.snuba_migrations.metrics.templates import (
-    COL_SCHEMA_DISTRIBUTIONS,
+    COL_SCHEMA_DISTRIBUTIONS_V3,
```

Comment:
as a general rule we try not to modify old migrations, can we just have the version suffix in the new column schema